mirdata/utils.py

```diff
@@ -29,11 +29,45 @@
 import hashlib
 import os
 import json
+import types
 
 
 MIR_DATASETS_DIR = os.path.join(os.getenv('HOME', '/tmp'), 'mir_datasets')
 
 
+class Track(object):
+    def __repr__(self):
```

Comment:
right now you're not truncating strings or anything right? if an object has a really long value it might hang ipython.

Reply:
good point - will add this